docs/index.md

```diff
@@ -1,639 +1,128 @@
 
-Flox installs packages into _environments_ as opposed to installing them into
-globally accessible directories in various places on your system.
-This allows you to have different versions of your tools across different
-projects without them interfering or conflicting.
+    ---
 
-`cd` into your project and initialize the environment with [`flox init`][init].
+    Install `flox` with `apt`, `yum`, `brew`, or a system installer to get your dev 
+    environment set up in minutes.
 
-For Python, Node.js, and Go projects this will display a prompt asking if you
-would like Flox to automatically do some language-specific setup for you.
+    [:octicons-arrow-right-24: Download & Install][install_flox]
 
-=== "Python"
+    [:octicons-arrow-right-24: Flox in 5 minutes][flox_5_minutes]
+    
+    [:octicons-arrow-right-24: Search packages in FloxHub][floxhub_packages]{:target="_blank"}
 
-    ```
-    $ cd my_project
+-   :material-nix:{ .lg .middle } __Already using Nix? Start here__
 
-    my_project $ flox init
-    Flox detected a Python project with the following Python provider(s):
+    ---
 
-    * latest python (requirements.txt)
+    Flox brings the power of Nix to your team, and can simplify your workflows too.
 
-      Installs latest python (3.12.5) with pip bundled.
-      Adds hooks to setup and use a venv.
-      Installs dependencies to the venv from: requirements.txt
+    [:octicons-arrow-right-24: Install Flox with flakes or profiles](install-flox.md?h=nix#__tabbed_1_6){:target="_blank"}
 
-    ! Would you like Flox to set up a standard Python environment?
-    You can always change the environment's manifest with 'flox edit'
-    > Yes - with latest python
-      No
-      Show suggested modifications for latest python
-    [Use '--auto-setup' to apply Flox recommendations in the future.]
-    ...select yes...
-    ✨ Created environment 'my_project' (aarch64-darwin)
-    ✅ 'python3' installed to environment 'my_project'
+    [:octicons-arrow-right-24: Flox brings Nix to your teams](https://flox.dev/blog/its-time-to-bring-nix-to-work/){:target="_blank"}
 
-    Next:
-      $ flox search <package>    <- Search for a package
-      $ flox install <package>   <- Install a package into an environment
-      $ flox activate            <- Enter the environment
-      $ flox edit                <- Add environment variables and shell hooks
-    ```
+    [:octicons-arrow-right-24: Use flakes in Flox](https://flox.dev/blog/extending-flox-with-nix-flakes/){:target="_blank"}
 
-    If your Python project contains a `requirements.txt`,
-    a Poetry `pyproject.toml`, or a generic `pyproject.toml`,
-    you'll be asked whether you'd like Flox to install Python, `pip`, and
-    Poetry (if applicable).
+</div>
 
-    If you accept, it will also create a virtual environment that's activated
-    automatically when you activate your Flox environment.
+Guides for getting started in just a few of the many languages Flox supports:
```

Comment:
My nitpick about the word "guides" here is that is sounds like the linked blog posts are the suggested way of getting started, when in actuality they're examples of projects (I haven't read all of the blog posts, but that's at least how my Rust post was). Maybe "Examples" instead?